app/connectors/SubscriptionCacheConnector.scala

```diff
@@ -27,7 +27,7 @@ class SubscriptionCacheConnector @Inject()(
                                                override val http: WSClient
                                              ) extends CacheConnector {
 
```

Comment:
behavior tests are missing for this class

Reply:
Added behaviour tests